_episodes/44.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: episode
+title: "44: Seeing clearly with Metabase"
+date: 2023-01-26
+tags: trino metabase visualization client clojure bi reporting
+youtube_id: ""
+wistia_id: ""
+sections:
+   - title: "Release update"
+     desc: "405 and 406"
+     time: 
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
+  [Metabase](https://www.metabase.com/)
+* [Andrew DiBiasio](https://www.linkedin.com/in/andrewdibiasio/), Software
+  Engineer at [Starburst](https://starburst.io)
+* [Piotr Leniartek](https://www.linkedin.com/in/piotrleniartek), Project Manager
+  at [Starburst](https://starburst.io)
+
+## Recap of Trino in 2022
+
+Highlights from the blog post [The rabbit reflects on Trino in 2022]({% post_url
+2023-01-10-trino-2022-the-rabbit-reflects %}):
+
+* 
+
+## Releases 404 to 406
+
+Official highlights from Martin Traverso:
+
+[Trino 404]({{site.url}}/docs/current/release/release-404.html) not found
+
+[Trino 405]({{site.url}}/docs/current/release/release-405.html)
+
+* Support for `ALTER COLUMN ... SET DATA TYPE` statement.
+* Support for Apache Arrow when reading from BigQuery.
+* Support for views in the Delta Lake connector.
+* Support for the Iceberg REST catalog.
+* Support for Protobuf encoding in the Kafka connector.
+* Support for fault-tolerant execution in the MongoDB connector.
+* Support for `DELETE` and query pushdown in the Redshift connector.
+* Performance improvements when reading Parquet data.
+
+[Trino 406]({{site.url}}/docs/current/release/release-406.html)
+
+* tbd
+
+Further thoughts and highlights from Cole:
+
+* tbd
+
+More detailed information is available in the release notes for
+[Trino 405]({{site.url}}/docs/current/release/release-405.html),
+and
+[Trino 406]({{site.url}}/docs/current/release/release-406.html).
+
+We also shipped trino-python-client 0.321.0 with the following improvements:
+
+* Add support for SQLAlchemy 2.0.
+* Add support for `varbinary` query parameters.
+* Add support for variable precision `datetime` types.
+
+
+
+
+
+## What is Metabase
+
+## History of Metabase and Trino
+
+
+## Core advantages of using Metabase with Trino
+
+connectint to database that are not supported by metabase
+
+row level secruity on any catalog 
+
+joining sql and no-sql databse (cross databse)
+
+## Demo of the episode: Metabase and Trino
+
+GitHub - paoliniluis/metabase-trino
+## Real world use cases at Meesho
+
+
+## PR of the episode
+
+starburstdata/metabase-driver: Starburst Metabase driver 
```

Comment:
Should we link https://github.com/starburstdata/metabase-driver

Reply:
Yeah .. all this is still a draft .. i will flesh out a lot .. mostly on Monday